updatecli/updatecli.d/git-lfs.yml

```diff
@@ -11,7 +11,6 @@ sources:
       token: "{{ requiredEnv .github.token }}"
       username: "{{ .github.username }}"
       versionFilter:
-        pattern: ~2
```

Comment:
Either you have to update this line and set it to `pattern: ~3` to ensure that the git-lfs version if always 3.x (and avoid breaking things when the 4.x we'll be there), either you should keep it deleted but set the attribute `kind` to `latest`